source/plugins/ruby/CAdvisorMetricsAPIClient.rb

```diff
@@ -304,6 +308,18 @@ def getContainerCpuMetricItems(metricJSON, hostName, cpuMetricNameToCollect, met
                         telemetryProps["multilineLanguages"] = @clusterMultilineLanguages
                       end
                     end
+                    if (!@clusterKubernetesMetadataEnabled.nil? && !@clusterKubernetesMetadataEnabled.empty?)
+                      telemetryProps["metadataEnabled"] = @clusterKubernetesMetadataEnabled
+                    end
```

Comment:
Why are we sending the telemetry from here? Shouldn't this all be sent as part of ContainerLog event or something? 
Also, if you decide to add the telemetry here, now that this is all moved to golang as well, you will have to add the telemetry there as well

Reply:
discussed with Ganga this is a valid point, will keep this and add the telemetry in go as well